src/components/mdxComponents/index.tsx

```diff
@@ -23,11 +23,14 @@ import SvgRenderer from './svg';
 import Command from './command';
 import Quote from './quote';
 import BlockQuote from './block-quote';
+import CustomTableCell from './custom-table-cell';
 
 export default {
   ...baseComponents,
   svg: (props: any) => <SvgRenderer {...props} />,
   quote: (props: any) => <Quote {...props} />,
   blockquote: (props: any) => <BlockQuote {...props} />,
   command: (props: any) => <Command {...props} />,
+  td: ({ align, ...props }) => <CustomTableCell {...props} align={align || undefined} />,
+
```

Comment:
why did this move from `baseComponents.tsx`?

Reply:
Cyclic dependency issue.